pkg/registry/manifest.go

```diff
@@ -48,6 +56,15 @@ func isManifest(req *http.Request) bool {
 	return elems[len(elems)-2] == "manifests"
 }
 
+func isTags(req *http.Request) bool {
+	elems := strings.Split(req.URL.Path, "/")
+	elems = elems[1:]
+	if len(elems) < 4 {
+		return false
+	}
+	return elems[len(elems)-2] == "tags"
```

Comment:
This is _probably_ fine, but checking that it ends with `/tags/list` is probably safer.